src/models/library_with_filters.rs

```diff
@@ -315,6 +368,11 @@ fn catalog_update<F: LibraryFilter>(
             .filter(|library_item| match &selected.request.r#type {
                 Some(r#type) => library_item.r#type == *r#type,
                 None => true,
+            })          
+            .filter(|library_item| match &selected.request.watched {
+                Watched::NotWatched => !library_item.watched(),
+                Watched::Watched => library_item.watched(),
+                Watched::Any => true,
             })
             .sorted_by(|a, b| match &selected.request.sort {
                 Sort::LastWatched => b.state.last_watched.cmp(&a.state.last_watched),
```

Comment:
And add the filter logic of  `Sort:Watched` here

Reply:
I'm not interested in sorting, I'm interested ina filter to reduce the amount of items shown in the library. By making the default not watched, it's really easy to keep it clean and only see new things on it.

Reply:
But I'm not sure if you are suggesting to add the options in sort but still be able to do a filter?

Reply:
Keeping it separated also helps to be able to sort and filter at the same time, so we don't lose features

Reply:
Ok i misunderstood then, can you rename `Watched` to `Filter` instead ?

Reply:
Sounds good